### PR TITLE
Dev 2

### PR DESCRIPTION
**Pull request by Kamil-Sabbagh**

### CODE REVIEW

**Comment by ilov3** [comment]
Hey @Kamil-Sabbagh! Seems like there is some issue with your PR - it reports about 3830 files changed and 665347 new lines of code which is weird. Could you please re-check on your side what was pushed into the repo?

**Comment by Kamil-Sabbagh** [comment]
> Hey @Kamil-Sabbagh! Seems like there is some issue with your PR - it reports about 3830 files changed and 665347 new lines of code which is weird. Could you please re-check on your side what was pushed into the repo?

I have removed some files, you can check it now.

File: Scrapping a Website part 2/scrapping_a_website/scrapping_a_website/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,241 @@
+aiofiles==0.4.0
```
Comment (ilov3) [inline Scrapping a Website part 2/scrapping_a_website/scrapping_a_website/requirements.txt]: Too many dependencies listed here, are you sure that all of them needed?

File: Scrapping a Website part 2/scrapping_a_website/scrapping_a_website/spiders/main.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import scrapy
```
Comment (ilov3) [inline Scrapping a Website part 2/scrapping_a_website/scrapping_a_website/spiders/main.py:1]: There are a lot of violations of PEP-8 convention. Could you please fix them?

**Review by ilov3 — COMMENTED** [review]
Thanks for updating your repo. Btw could you please also exclude `*.pyc` files from it?

**Pull request merged**